ansible/roles/nginx/vars/icds_tableau.yml

```diff
@@ -0,0 +1,20 @@
+---
+nginx_sites:
+- server:
+   file_name: icds_tableau
+   listen: "443 ssl"
+   # Move these to a localsetting?
+   server_name: icds.commcarehq.org
+   error_page: "502 503 /errors/50x.html"
+   proxy_set_headers:
+   - "Host $http_host"
+   - "X-Forwarded-For $remote_addr"
+   - "X-Forwarded-Proto $scheme"
+   - "X-Forwarded-Host $host:443"
+   location1:
+    name: /
+    proxy_pass: http://tableau2.internal.commcarehq.org
```

Comment:
Sounds like I need to make a note (on the wiki) of which machine is which. tableau2 is ICDS, tableau3 is Ebola.